rcl/src/rcl/publisher.c

```diff
@@ -94,10 +94,10 @@ rcl_publisher_init(
 rcl_ret_t
 rcl_publisher_fini(rcl_publisher_t * publisher, rcl_node_t * node)
 {
+  if (!rcl_publisher_is_valid(publisher)) return RCL_RET_PUBLISHER_INVALID;
+  else if (!rcl_node_is_valid(node)) return RCL_RET_NODE_INVALID;
```

Comment:
We always use `{}` to enclose if like statements. You'll want to run our linters over this code to catch any other issues like this. You can either run the tests or you can just run the linters directly:

```
$ ament_uncrustify /path/to/ws/src/ros2/rcl
$ ament_cpplint /path/to/ws/src/ros2/rcl
```

Uncrustify can also automatically fix the style with the `--reformat` option.